handlers/main.yml

```diff
@@ -1,2 +1,13 @@
 ---
 # handlers file for etcd-cluster
+
+- name: restart etcd
+  become: yes
+  become_user: root
+  when: etcd_launch | bool
+  service:
+    name: etcd
+    state: restarted
+  run_once: yes
+  loop: '{{ groups[etcd_master_group_name] }}'
```

Comment:
what about the non-voting members?

Reply:
I've added one more handler for non-voting members